woocommerce/woocommerce-gutenberg-products-block
# Remove RawHTML used for notice translations

Blocked by: #888.

Coming from https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/884#discussion_r315385440. We need to remove calls to `<RawHTML>` that were used to interpolate HTML tags with translations.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/63340006-661fd500-c346-11e9-8f06-d9a1938edd38.png)

### How to test the changes in this Pull Request:

1. In _WooCommerce_ > _Settings_ > _Products_ > _General_ > _Product ratings_ uncheck _Enable star rating on reviews_.
2. In _Settings_ > _Discussion_ > _Avatar Display_ uncheck _Show Avatars_.
3. Create a post with a _Reviews by Product_ block and verify notices are displayed correctly.

### Discussion
~This PR makes translation strings quite ugly and difficult to localize: `Product rating is disabled in your` and `store settings` are separate strings.~

@joshuatf ~how does it sound rewritting those notices to something like `Product rating is disabled in your store settings. <Read more>` or `Product rating is disabled in your store settings. <Modify them>` so the link is not in the middle of the sentence?~
